api/types/stores.go

```diff
@@ -17,6 +17,7 @@ type SockStore interface {
 	CountSocks() (int, error)
 	GetSockByID(sockID int) (*Sock, error)
 	GetSockVariants(sockID int) ([]SockVariant, error)
+	UpdateSock(sockID int, sock Sock, variants []SockVariant) error
```

Comment:
Add the signature for checking if a sock variant exists: `SockVariantExists(sockID int, size string) (bool, error)`

Reply:
Fixed!!